### PR TITLE
Consolidate the external API of all service objects

### PR DESCRIPTION
**Pull request by teoljungberg**
- Don't duplicate the methods signature in static facades

  Since the method signature, including its parameters are already
  present in the initializer. We can safely remove it from the static
  facade without hurting readability.

- Rename `#run` to `#call`.
- Rename `.run` to `.call`.
- Add static facades where they are missing.

### CODE REVIEW

File: app/services/build_config.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class BuildConfig
-  def self.for(hound_config:, name:, owner:)
-    new(hound_config: hound_config, name: name, owner: owner).config
+  def self.for(*args)
```
Comment (gylaz) [inline app/services/build_config.rb]: This `for` really should just be a `call`.

File: app/queries/recent_builds_by_repo_query.rb [hunk 2]
```diff
@@ -1,15 +1,15 @@
 class RecentBuildsByRepoQuery
   NUMBER_OF_BUILDS = 20
 
-  def self.run(*args)
-    new(*args).run
+  def self.call(*args)
+    new(*args).call
```
Comment (gylaz) [inline app/queries/recent_builds_by_repo_query.rb]: What are your thoughts on using [`static_facade`](https://github.com/barsoom/attr_extras#static_facade) from `attr_extras` gem?

Comment (teoljungberg) [inline app/queries/recent_builds_by_repo_query.rb]: I like it!

File: app/services/build_config.rb (hunk 1 above)
Comment (teoljungberg) [inline app/services/build_config.rb]: Good catch!

**Pull request merged**